descriptors/ModuleDescriptor-template.json

```diff
@@ -1367,6 +1367,11 @@
           "pathPattern": "/inventory-storage/migrations/jobs/{id}",
           "permissionsRequired": ["inventory-storage.migration.job.item.get"]
         },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/inventory-storage/migrations/jobs",
+          "permissionsRequired": ["inventory-storage.migration.job.item.get"]
```

Comment:
Is it a general practice to share permission between `collection` and `item`?

Reply:
No, but for admin API it makes sense. Too many unnecessary permissions can slow down the current login approach on the folio. It already affects karate tests, so I decided to avoid creating new permission for the same retrieve operations in the admin API